examples/navigation/epics/adminAccess.js

```diff
@@ -0,0 +1,15 @@
+import { push, LOCATION_CHANGE } from 'react-router-redux';
+
+function hasAccess() {
+  return false;
+}
+
+export default function adminAccess(action$) {
+  return action$.ofType(LOCATION_CHANGE)
+    .map(action => action.payload.pathname)
+    .filter(pathname => pathname === '/admin')
+    .map(hasAccess)
+    .filter(access => !access)
```

Comment:
I would omit the lines of code, but add a comment telling the reader what they _could_ do there.